web/sites/default/config/clamav.settings.yml

```diff
@@ -0,0 +1,15 @@
+enabled: true
```

Comment:
@brittanysmart , this should not be here. It means you did a `lando cex` after your `lando latest` and before you did your `lando refresh`. That caused the local config to get overridden with the dev config.

Reply:
Should I update to false?

Reply:
The best way to fix this order of operations error is to run `git checkout release web/sites/default/config/clamav.settings.yml` to revert the change. Editing any Drupal generated YML file is bad practice. It introduces the possibility of configuration corruption. I try to avoid it when possible. 😄

Reply:
I should note, that when you do that, I believe it will delete this file as clamav settings only exist in the dev config split and up.

Reply:
Thanks for including the command, but it did not work because this file doesn't exist or isn't tracked in git. I ended up deleted this file and committing.